.github/workflows/publish.yml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Publish Release
+
+on:
+  # Only run the workflow when a PR is merged to the master branch.
+  pull_request:
+    branches: master
+    types: closed
```

Comment:
If we close a PR without merging would that also trigger this?

Reply:
It does, but then will get immediately skipped because of:

```
if: github.event.pull_request.merged
```